MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -174,7 +174,7 @@ protected internal override object Deserialize(IntermediateReader input, Content
                             continue;
                         
                         // We failed to find a required element.
-                        throw new InvalidContentException(string.Format("The Xml element `{0}` is required!", info.Attribute.ElementName));
+                        throw new InvalidContentException(string.Format("The Xml element `{0}` is required, But found '{1}' instead, try swappping elements as they may be out of order, or required elements may be missing altogether", info.Attribute.ElementName, input.Xml.Name));
```

Comment:
Some grammar fixes:

```
The Xml element `{0}` is required, but element `{1}` was found.  Try changing the element order or adding missing elements.
```

Reply:
Do you want the line number as well?
```
The Xml element `{0}` is required, but element `{1}` was found at line {2}. Try changing the element order or adding missing elements.", info.Attribute.ElementName, input.Xml.Name,((IXmlLineInfo)input.Xml).LineNumber);
```
Or shorter wording with position:
```
The Xml element `{0}` is required, but element `{1}` was found at {2},{3}. Try changing the element order or adding missing elements.", info.Attribute.ElementName, input.Xml.Name,((IXmlLineInfo)input.Xml).LineNumber, ((IXmlLineInfo)input.Xml).LinePosition);
```